src/MergeContext.js

```diff
@@ -891,14 +876,53 @@ class PullRequest {
 
     _toString() {
         let str = this._debugString();
-        if (this._tagSha !== null)
-            str += ", tag: " + this._tagSha.substr(0, this._shaLimit);
+        if (this._stagedSha())
+            str += ", staged: " + this._stagedSha().substr(0, this._shaLimit);
         return str + ")";
     }
 
+    _dateForDaysAgo(days) {
+        let d = new Date();
+        d.setDate(d.getDate() - days);
+        const dd = String(d.getDate()).padStart(2, '0');
+        const mm = String(d.getMonth() + 1).padStart(2, '0'); //January is 0!
+        const yyyy = d.getFullYear();
+        return yyyy + '-' + mm + '-' + dd;
+    }
+
+    /// Checks whether the PR base branch has this PR's staged commit merged.
+    /// TODO: for now the search scope of this method is limited only by
+    /// the project 'default branch', configured on GitHub, which is usually
+    /// (but not always) 'master' branch.
+    async _mergedSomeTimeAgo() {
+        const query = 'repo:' + Config.owner() + "/" + Config.repo() + '+#' + this._prNumber() +
```

Comment:
Can we add parenthesis and a space for the full ` (#NNN)` pattern? If not, we should probably add a comment why not.

Reply:
Done.